src/index.js

```diff
@@ -20,4 +21,4 @@ const styled = _styled(_styledComponent(_ComponentStyle(generateAlphabeticName))
 
 /* Export everything */
 export default styled
-export { css, keyframes, injectGlobal, ThemeProvider }
+export { css, keyframes, injectGlobal, ThemeProvider, serverStyleSheet }
```

Comment:
Isn't it better to export it as `ServerStyleSheet` (pascal case)? What do you think?